tests/yam/migration/migration_unattended.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Migration activation then reboot to perform migration.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use grub_utils 'grub_test';
+use migration 'disable_installation_repos';
+use power_action_utils 'power_action';
+use utils 'zypper_call';
+
+sub await_migration {
+    my $timeout = 600;
```

Comment:
this is too much, probably a bug if it would take so long.

Reply:
I don't remember it is so slow, be more conservative according the job runs and we will adjust it later, might take less than an installation, but not sure.